src/components/tests/content-tab-assistant.test.tsx

```diff
@@ -183,4 +188,47 @@ describe( 'ContentTabAssistant', () => {
 
 		expect( screen.queryByText( 'New message' ) ).toBeNull();
 	} );
+
+	test( 'does not render the Welcome messages and example prompts when not authenticated', () => {
+		( useAuth as jest.Mock ).mockImplementation( () => ( {
+			client: {
+				req: {
+					post: clientReqPost,
+				},
+			},
+			isAuthenticated: false,
+			authenticate,
+		} ) );
+		render( <ContentTabAssistant selectedSite={ runningSite } /> );
+		expect( screen.getByText( 'Hold up!' ) ).toBeInTheDocument();
+		expect( screen.queryByText( 'Welcome to our service!' ) ).not.toBeInTheDocument();
+	} );
+
+	test( 'confirm that Welcome messages and example prompts are present when the conversation is started', () => {
+		render( <ContentTabAssistant selectedSite={ runningSite } /> );
+		expect( mockFetchWelcomeMessages ).toHaveBeenCalledTimes( 1 );
+		expect( screen.getByText( 'Welcome to our service!' ) ).toBeInTheDocument();
+		expect( screen.getByText( 'How to create a WordPress site' ) ).toBeInTheDocument();
+		expect( screen.getByText( 'How to clear cache' ) ).toBeInTheDocument();
+		expect( screen.getByText( 'How to install a plugin' ) ).toBeInTheDocument();
+	} );
+
+	test( 'confirm that Welcome messages are present, the selected prompt is present, other prompts are removed', async () => {
+		render( <ContentTabAssistant selectedSite={ runningSite } /> );
+
+		expect( await screen.findByText( 'Welcome to our service!' ) ).toBeInTheDocument();
+		expect( await screen.findByText( 'How to create a WordPress site' ) ).toBeInTheDocument();
+		expect( await screen.findByText( 'How to install a plugin' ) ).toBeInTheDocument();
+
+		const samplePrompt = await screen.findByRole( 'button', {
+			name: 'How to create a WordPress site',
+		} );
+		expect( samplePrompt ).toBeInTheDocument();
+		fireEvent.click( samplePrompt );
+
+		// Check if the selected prompt is still present and other prompts are removed
+		expect( screen.getByText( 'Welcome to our service!' ) ).toBeInTheDocument();
+		expect( screen.getByText( 'How to create a WordPress site' ) ).toBeInTheDocument();
+		expect( screen.queryByText( 'How to clear cache' ) ).not.toBeInTheDocument();
```

Comment:
We could also include to check the other prompt:
```suggestion
		expect( screen.queryByText( 'How to clear cache' ) ).not.toBeInTheDocument();
		expect( screen.queryByText( 'How to install a plugin' ) ).not.toBeInTheDocument();
```

Reply:
Added in https://github.com/Automattic/studio/pull/293/commits/fb86ecb2ce764ef78754ec37483a838dc30f5197